Sources/LiveViewNative/Coordinators/LiveConnectionError.swift

```diff
@@ -53,7 +53,7 @@ public enum LiveConnectionError: Error, LocalizedError {
             case .document:
                 return "document"
             case .csrfToken:
-                return #"<meta name="csrf-token">"#
+                return #"<.csrf_token />"#
```

Comment:
what is this doing?

Reply:
This is used as part of the error message when the CSRF token is missing.